src/Config.js

```diff
@@ -100,6 +148,29 @@ module.exports = class extends ChainedMap {
               .forEach(name => this.plugin(name).merge(value[name]));
           }
 
+          /**
+           * @see ./Plugin
+           * merge an array of plugins
+           */
+          case 'plugins': {
+            if (Array.isArray(value)) {
```

Comment:
https://github.com/mozilla-neutrino/webpack-chain/pull/21#issuecomment-289888748
if I merge in a config of existing plugins, it does not merge properly

by plugin interface I mean if there is a plugin that results from a `.toConfig`, or, from merging in an existing webpack config